doc/source/api.rst

```diff
@@ -1617,7 +1617,6 @@ IntervalIndex Components
    IntervalIndex.from_arrays
    IntervalIndex.from_tuples
    IntervalIndex.from_breaks
-   IntervalIndex.from_intervals
```

Comment:
Should this be removed from api.rst now, or should I wait until `from_intervals` is fully removed and not just deprecated?

Reply:
yes this is fine